client/src/lifecycleManager.js

```diff
@@ -104,17 +104,11 @@ class LifecycleManager extends LuigiClientBase {
 
     helpers.addEventListener('luigi.navigate', e => {
       setContext(e.data);
-      if (!this.currentContext.internal.isNavigateBack && !this.currentContext.withoutSync) {
+      const withoutSync = this.currentContext.withoutSync === true;
+      if (!this.currentContext.internal.isNavigateBack && !withoutSync) {
```

Comment:
I think we should avoid double negative boolean logic like this. e.g `!withoutSync`
Are we able to slightly improve it?

Reply:
true, refactored it 👍